### PR TITLE
Update v1.2.1

### PR DESCRIPTION
**Pull request by EddyIV**
Updated the following to Greymatter v1.2.1 server versions:
* gm-proxy
* gm-jwt-security
* gm-cli
* gm-dashboard
* gm-slo

***This branch currently has the changes from the k3d-v3 branch (which has not been merged), as I'm using k3d 3.0.0.***

### CODE REVIEW

File: docs/Deploy with K3d.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ## Local Usage
 
 1. `make k3d` - start a kubernetes cluster locally.
-2. `export KUBECONFIG="$(k3d get-kubeconfig --name='greymatter')"` - configures `kubectl` to use the local k3d cluster.
```
Comment (whenhellfreezes) [inline docs/Deploy with K3d.md:11]: The more general version here is better because I think you are not gaurenteed that the config will be at this path.

**Comment by justincely** [comment]
@chrisbsmith @zoemccormick is this OBE?

**Comment by chrisbsmith** [comment]
@justincely My guess is "yes", but we need to put eyes on it to confirm

**Comment by zoemccormick** [comment]
@justincely @chrisbsmith yes I think this can be closed

**Pull request closed without merging**